packages/typescript-estree/tests/lib/createProjectService.test.ts

```diff
@@ -1,15 +1,16 @@
 import debug from 'debug';
```

Comment:
Side note (not this PR) - we should probably build out a test case using `typescript-estree/tests/fixtures` that allows us to do tests with project service without mocking out the entire world.

Reply:
I started to do this but I think the consensus was it fit more into integration: https://github.com/typescript-eslint/typescript-eslint/tree/148a9d0f7c670a74152279974988e9107e4be2f7/packages/typescript-estree/tests/fixtures/projectServicesComplex.

I know there is a desire to release to v8 soon.  I don't have a ton of time this week.  @JoshuaKGoldberg and @bradzacher, should I just open a new issue to follow back up on this?

Reply:
A new issue would be very reasonable, yes please!

Reply:
Opened https://github.com/typescript-eslint/typescript-eslint/issues/9641